README.md

```diff
@@ -40,3 +40,22 @@ zsh-copilot --help
 
 Type in your command or your message and press `CTRL + Z` to get your suggestion!
 
+## Ollama Llama3 usage
+
+### Configuration
+
+You need to have Ollama and llama3 installed : 
+
+```sh
+curl -fsSL https://ollama.com/install.sh | sh
+sudo systemctl start ollama
+ollama pull llama3.1
+```
+
+### Set Ollama active
+
+Modify the **zsh-copilot.plugin.zsh** file to set the shortcut  of your choice like this : 
```

Comment:
`If there is no OPENAI_API_KEY nor any ollama running, just display an error when trying the shortcut ?`

Yes just an error is displayed

Reply:
`I'm not sure asking the user to modify the file to set the shortcut is the best way.`

I think is a good idea because that permits people to choose they favorite shortcut to use the AI,